lib/commons/dom/is-hidden-with-css.js

```diff
@@ -0,0 +1,50 @@
+/* global dom */
+
+/**
+ * Determine whether an element is hidden based on css
+ * @method isHiddenWithCSS
+ * @memberof axe.commons.dom
+ * @instance
+ * @param {HTMLElement} el The HTML Element
+ * @param {Boolean} descendentVisibilityValue (Optional) immediate descendant visibility value used for recursive computation
+ * @return {Boolean} the element's hidden status
+ */
+dom.isHiddenWithCSS = function isHiddenWithCSS(el, descendentVisibilityValue) {
```

Comment:
I like the new name for this method. 👍 

Are you pulling any of the code out of is-visible, or is that mostly duplicate at this point?

Reply:
This has some replication from `is-visible` and `is-hidden`. We may need to do some refactor afterwards.